docs/source/configuration/overview.mdx

```diff
@@ -403,6 +403,21 @@ See [Tracing in the Apollo Router](./tracing/).
 Automatic Persisted Queries (APQ) enable GraphQL clients to send a server the _hash_ of their query string, _instead of_ the query string itself. This can significantly reduce network usage for very large query strings.
 The Apollo Router automatically supports APQ via its in-memory cache. See the [caching documentation](./caching) for related options.
 
+Subgraph requests support automatic persisted queries) by default. It can be deactivated with the `apq` option:
```

Comment:
```suggestion
Subgraph requests from the Router will attempt to use APQ by default.  Disabling APQ on a subgraph can be done either globally or on a per-subgraph request.   The following example shows how to use a mixture of both global and per-subgraph configuration to accomplish enabling APQ on the `products` subgraph, while otherwise disabling it:
```